### PR TITLE
Java 11.2: Overloading Vararg Methods #2218

### PR DESCRIPTION
**Pull request by Harsh-Chaudhary7**
<hr>

## Description 📜

Explanation of Method Overloading in Varargs #2218 

<hr>

## Type of change 📝

- [x] Doc#umentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Java_#Domain

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Harsh-Chaudhary7** [comment]
Please review it @AkMo3

**Review by garvitraj — CHANGES_REQUESTED** [review]
Capture

Pls do the needful changes

**Comment by garvitraj** [comment]
@AkMo3 pls look into it once

**Comment by Harsh-Chaudhary7** [comment]
> Capture
> 
> Pls do the needful changes

I am sorry i couldn't understand what changes are you asking me to do. Can you please elaborate a little?

**Review by sumitaccess007 — CHANGES_REQUESTED** [review]
@Harsh-Chaudhary7,
Following are some review points, please make necessary changes -

1. In the beginning write simple explanation about "**varargs method**", what it is.
2. Write its Syntax separately, "Syntax Of Varargs"
3. "**Relation among Primitive Data Types**", is written using HTML Lists, appears as raw, they are not recognized as lists.
4. It will be good if you can also write about internal working functionality of varargs, means How JVM treats it during program execution.

**Comment by sumitaccess007** [comment]
> > Capture
> > Pls do the needful changes
> 
> I am sorry i couldn't understand what changes are you asking me to do. Can you please elaborate a little?
@Harsh-Chaudhary7 - You are trying to create a Unordered List here using HTML List, but unfortunately they are not appearing as list, list tags are appearing along with relations between primitive data types.
You can use "-" to make them a Unordered List in .md file.

**Comment by Harsh-Chaudhary7** [comment]
> @Harsh-Chaudhary7,
> Following are some review points, please make necessary changes -
> 
> 1. In the beginning write simple explanation about "**varargs method**", what it is.
> 2. Write its Syntax separately, "Syntax Of Varargs"
> 3. "**Relation among Primitive Data Types**", is written using HTML Lists, appears as raw, they are not recognized as lists.
> 4. It will be good if you can also write about internal working functionality of varargs, means How JVM treats it during program execution.

I'll make the neccesary changes for the 3rd point but for the other 3 points there are already issues created and files are present there. So, do i have to include these points too?

**Comment by sumitaccess007** [comment]
> > @Harsh-Chaudhary7,
> > Following are some review points, please make necessary changes -
> > 
> > 1. In the beginning write simple explanation about "**varargs method**", what it is.
> > 2. Write its Syntax separately, "Syntax Of Varargs"
> > 3. "**Relation among Primitive Data Types**", is written using HTML Lists, appears as raw, they are not recognized as lists.
> > 4. It will be good if you can also write about internal working functionality of varargs, means How JVM treats it during program execution.
> 
> I'll make the neccesary changes for the 3rd point but for the other 3 points there are already issues created and files are present there. So, do i have to include these points too?

Then it's okay, you can proceed with 3rd point. Also take care of below point.
Keep simple file name, remove Issue Id, topic number from filename. You can simply put it like "Overloading_Vararg_Methods.md"

File: Java/Varargs/Overloading_Vararg_Methods/Java 11.2: Overloading Vararg Methods #2218.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+   <li>int > char</li>
+    
+   <li>int > short</li>
+    
+   <li>short > byte</li>
+   
+   </ul>
+    
+    
+    So the output of this code snippet will be **_int varargs_**, because int is more specific than double and float.
```
Comment (sumitaccess007) [inline Java/Varargs/Overloading_Vararg_Methods/Java 11.2: Overloading Vararg Methods #2218.md:73]: Update this part as well, it is appearing as **_int varargs_**.

Comment (Harsh-Chaudhary7) [reply Java/Varargs/Overloading_Vararg_Methods/Java 11.2: Overloading Vararg Methods #2218.md]: Done! Anything else?

Comment (sumitaccess007) [reply Java/Varargs/Overloading_Vararg_Methods/Java 11.2: Overloading Vararg Methods #2218.md]: Looks fine to me now.
@garvitraj - Please have a final look.
Thanks

**Comment by sumitaccess007** [comment]
Looks fine to me now.
@garvitraj - Please have a final look.
Thanks

**Review by garvitraj — APPROVED** [review]
Ya ig its fine 
thanks @sumitaccess007 for reviw

**Pull request merged**